src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs

```diff
@@ -4317,13 +4399,7 @@ private SqlParameter GetSqlParameterWithQueryText(string queryText)
             return sqlParam;
         }
 
-        /// <summary>
-        /// Constructs the sp_describe_parameter_encryption request with the values from the original RPC call.
-        /// Prototype for <sp_describe_parameter_encryption> is 
-        /// exec sp_describe_parameter_encryption @tsql=N'[SQL Statement]', @params=N'@p1 varbinary(256)'
-        /// </summary>
-        /// <param name="originalRpcRequest">Original RPC request</param>
-        /// <param name="describeParameterEncryptionRequest">sp_describe_parameter_encryption request being built</param>
+       
```

Comment:
Plz don't delete any documentation, this is private method, not harming anything.

Reply:
probably we will end up having to summaries and params if the compiler does not complain about it. we already have params in the include file